src/commands/backend/metrics.ts

```diff
@@ -0,0 +1,48 @@
+import { Command } from '@oclif/core'
+import { Jamsocket } from '../../jamsocket'
+import chalk from 'chalk'
+
+function prettifyBytes(bytes: number) {
+  const suffixes = ['B', 'KiB', 'MiB', 'GiB']
+  let i = 0
+  for (i; bytes > 1 << 10; i++) {
+    bytes /= 1 << 10
+  }
+  return bytes.toFixed(2) + ' ' + suffixes[i]
+}
+
+type BackendMetrics = {
+  cluster: string,
+  backend_id: string,
+  mem_used: number,
+  mem_available: number,
+  cpu_used: number,
+  sys_cpu: number
+}
+
+const formatToHeader = (a: string, header: string) => a.padEnd(header.length, ' ')
+export default class Metrics extends Command {
+  static description = 'Stream metrics from a running backend'
+  static examples = [
+    '<%= config.bin %> <%= command.id %> f7em2',
+  ]
+
+  static args = [
+    { name: 'backend', description: 'The name of the backend, a random string of letters and numbers returned by the spawn command.', required: true },
+  ]
+
+  public async run(): Promise<void> {
+    const jamsocket = Jamsocket.fromEnvironment()
+    const { args } = await this.parse(Metrics)
+    this.log(chalk.bold('cpu util\tmem used\tmem avail'))
+
+    await jamsocket.streamMetrics(args.backend, line => {
+      const metrics: BackendMetrics = JSON.parse(line)
+      const cpu_util = (metrics.cpu_used / metrics.sys_cpu) * 100
+      process.stdout.write(
+        `\r${formatToHeader(cpu_util.toFixed(1) + '%', 'cpu util')}\t` +
```

Comment:
If the line printed out is shorter than the line that was printed for the message before it, will some of the characters of the previous line not get overwritten? If so, we could either (1) end each line with a tab or (2) use `\033[2K\r` which I believe erases the line the cursor is on before doing the carriage return. Either way, it probably makes sense to leave a comment about what exactly we're doing since those escape sequences aren't obvious.